src/md/verification.md

```diff
@@ -36,14 +36,14 @@ Verify integrity of tar.gz file:
 ```bash
 gpg --verify $APEX_RELEASE_CANDIDATE-source-release.tar.gz.asc
 md5sum --check $APEX_RELEASE_CANDIDATE-source-release.tar.gz.md5
-sha512sum --check $APEX_RELEASE_CANDIDATE-source-release.tar.gz.sha
+sha512sum --check $APEX_RELEASE_CANDIDATE-source-release.tar.gz.sha512
```

Comment:
please change `sha512sum` to `shasum -a 512`.

Reply:
done